docs/source/locale/es/LC_MESSAGES/ipython.po

```diff
@@ -3,70 +3,80 @@
 # This file is distributed under the same license as the Jupyter
 # Documentation package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, 2019.
-#
+# 
+# Translators:
+# Valery Briz <valerybriz@gmail.com>, 2019
+# 
 #, fuzzy
 msgid ""
 msgstr ""
 "Project-Id-Version: Jupyter Documentation 4.1\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2019-07-04 19:51-0400\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"PO-Revision-Date: 2019-07-27 11:59+0000\n"
+"Last-Translator: Valery Briz <valerybriz@gmail.com>, 2019\n"
+"Language-Team: Spanish (https://www.transifex.com/project-jupyter/teams/100899/es/)\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=utf-8\n"
+"Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: Babel 2.6.0\n"
+"Language: es\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
 # e06aa197a71b41eeaf94dfe9f0caaca2
 #: ../../source/ipython/content-ipython.rst:2
 msgid "IPython"
-msgstr ""
+msgstr "IPython"
 
 # 93415afba3654a1c904000d9a30c7b04
 #: ../../source/ipython/content-ipython.rst:5
 msgid "Contents"
-msgstr ""
+msgstr "Contenidos"
 
 # 0fb8abaae6d6439eb2d541d1cc83182b
 #: ../../source/ipython/content-ipython.rst:13
 msgid "Description"
-msgstr ""
+msgstr "Descripción"
 
 # 0ffa752b43e04a8f84cf0772fdeeda3b
 #: ../../source/ipython/content-ipython.rst:15
 msgid "IPython provides a rich architecture for interactive computing with:"
 msgstr ""
+"IPython provee una rica arquitectura para computación interactiva con:"
 
 # c81ecc03613b4623a05ae0b069bfc6a8
 #: ../../source/ipython/content-ipython.rst:17
 msgid "A powerful interactive shell."
-msgstr ""
+msgstr "Una poderosa shell interactiva."
```

Comment:
I also have a doubt here, should we let it as "shell" or traduce it to "consola" or another spanish word like "terminal", etc? I checked this discusion: https://github.com/jupyterhub/the-littlest-jupyterhub/issues/188 but I didn't find it clear the final decision.

Reply:
@valerybriz I'm definitely not the one to answer here. Would you like me to leave the PR open until someone comments or merge it as-is?

Reply:
Don't worry today I translated some files that actually specified (in english) "shell" and "terminal" so I think we can difference those from the translations in spanish which are also specific. you can merge it thanks!